specification/liftrneon/Neon.Postgres.Management/tspconfig.yaml

```diff
@@ -14,6 +14,13 @@ options:
     azure-resource-provider-folder: "resource-manager"
     output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/neon.json"
     examples-dir: "{project-root}/examples"
+  "@azure-tools/typespec-python":
+    package-dir: "azure-mgmt-neonpostgres"
+    package-name: "{package-dir}"
+    flavor: "azure"
+    generate-test: true
+    generate-sample: true
+    examples-dir: "{project-root}/examples"
```

Comment:
we don't need `examples-dir` anymore. Here is template: https://github.com/Azure/azure-rest-api-specs/blob/10163f4114a84ea4e5281415277b958d18fd1ced/specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml#L19-L24

Reply:
@msyyc, updated